NASA-PDS/harvest
# Boolean and Date fields normalization. Harvest loads data into registry.

## 🗒️ Summary
* Boolean fields normalization (1/0 = true/false) based on LDD field definition
* Date fields normalization (convert to ISO format) based on LDD field definition
* Harvest loads data into registry directly. (Before, Registry Manager loaded data).

## ⚙️ Test Data and/or Report

Build Instructions:
* Build and install commons project first ("bool" branch)
* Build this project.

Test Instructions:
See related issues for boolean and date fields examples.
General instructions:
* Create new registry with registry manager.
* Harvest some data. 
* Check that data is loaded into Elasticsearch by Harvest tool (without running any registry manager commands).

## ♻️ Related Issues
https://github.com/NASA-PDS/harvest/issues/78
https://github.com/NASA-PDS/harvest/issues/54
